week2-section1/exercise1/Main.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+* Exercise 1 - Understanding classes
+Create a class called FirstClass with a public constructor and destructor. Have the constructor output �I�m in the constructor!� and have the destructor output �I�m in the destructor!�. Then, instantiate 5 classes and run your program, are the constructor and destructor calls where you expected them to be?
+
+Now create a loop of at least five iterations. For each iteration, instantiate your class once. Run your program, are the constructor and destructor calls where you expected them to be?
+*/
+
+#include <iostream>
+
+class FMyFirtsClass
```

Comment:
Nice for following convention! But be careful with your spelling `First`, not `Firts`, and follow the exercise's instructions carefully. The class should be called `FFirstClass` not `FMyFirstClass`. `My` generally doesn't add any value to a name.